### PR TITLE
ci: bump alpha version to prevent colisions

### PR DESCRIPTION
**Pull request by MarcMcIntosh**
## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] All new and existing tests are passing.

### CODE REVIEW

File: packages/init/src/steps/set-version.ts [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
 
 const defaultSliceMachineVersion = "0.0.41";
 
+function isRecord(o: unknown): o is Record<string, string> {
+  return typeof o === "object" && o !== null && !Array.isArray(o);
+}
+
 const getTheSliceMachineVersionInstalled = (
-  packageJson: NodeUtils.FileContent<NodeUtils.JsonPackage>
+  packageJson: null | NodeUtils.JsonPackage
```
Comment (Heyrows) [inline packages/init/src/steps/set-version.ts]: really weird that this has to be modified in this PR for the lint :x

File: packages/slice-machine/lib/io/Types.ts [hunk 2]
```diff
@@ -36,11 +41,30 @@ const generateAndWriteTypes = (env: BackendEnvironment) => {
...
 };
 
+export function getFromPackage(
+  key: "dependencies" | "devDependencies",
+  packageJson: JsonPackage | null
+): Record<string, string> {
+  if (
+    packageJson !== null &&
+    isObject(packageJson) &&
+    has(key, packageJson) &&
```
Comment (Heyrows) [inline packages/slice-machine/lib/io/Types.ts]: key in packageJson would also have worked in pure typescript :)

File: packages/slice-machine/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "slice-machine-ui",
-  "version": "0.5.1-dev.1",
+  "version": "0.5.1-dev.14",
```
Comment (Heyrows) [inline packages/slice-machine/package.json:3]: you didn't change the versions of the core and Clients in the dependencies to match the new versions.

File: packages/init/package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@slicemachine/init",
-  "version": "1.1.14-dev.0",
+  "version": "1.1.14-dev.9",
```
Comment (Heyrows) [inline packages/init/package.json:3]: you didn't change the versions of the core and Clients in the dependencies to match the new versions.

File: packages/client/package.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@slicemachine/client",
-  "version": "1.1.14-dev.0",
+  "version": "1.1.14-dev.9",
```
Comment (Heyrows) [inline packages/client/package.json:3]: you didn't change the versions of the core in the dependencies to match the new versions.

**Review by Heyrows — CHANGES_REQUESTED** [review]

File: packages/slice-machine/lib/io/Types.ts (hunk 2 above)
Comment (MarcMcIntosh) [reply packages/slice-machine/lib/io/Types.ts]: I tried that one a few times with no luck :/

File: packages/init/src/steps/set-version.ts (hunk 1 above)
Comment (MarcMcIntosh) [reply packages/init/src/steps/set-version.ts]: yeah, it was really annoying i tried a lot things before narrowing the types

**Pull request merged**